Base/QTCLI/qSlicerCLIModule.cxx

```diff
@@ -102,6 +102,11 @@ vtkMRMLAbstractLogic* qSlicerCLIModule::createLogic()
     logic->DeleteTemporaryFilesOff();
     }
 
+  if (d->Desc.GetParameterDefaultValue("AllowMemoryTransfer") == "false")
```

Comment:
@jbvimort  That will be an other instance of the code to update from `GetParameterDefaultValue` to `GetParameterValue` when you are done with you changes